src/core/AnimatedSet.js

```diff
@@ -8,10 +9,22 @@ class AnimatedSet extends AnimatedNode {
 
   constructor(what, value) {
     super({ type: 'set', what: what.__nodeID, value: value.__nodeID }, [value]);
+    invariant(
```

Comment:
I'd like to make this check before `super`. I consider it important bc line before we're assuming it's an object (`what.__nodeID`). If it's `null` or `undefined` we're getting some useless message (`undefined is not an object`).

Reply:
That sounds quite reasonable, not sure If I did it intentionally though. I can totally check that out.